### ISSUE TITLE
Preferred area pr user

### ISSUE THREAD

**Issue by Langveisfra** [meeting-room-booking-system/mrbs-code#3525]
Opening MRBS for the first time og re-opening the first page gives the area with the lowest index number regardless of the onlogged users interests. 
It could be an option in the User registration page for selecting "Preferred area", showing the preferred area when the index page is requested.

Here the user could choose among all active areas in a drop-down menu.
If the user does not choose any preferred area, the index page url https://[mrbs]/index.php?area=
(with blank area) doesn't do any harm.

**Comment by antham1616** [comment]
Hi
in your config.inc file you can just set the defaults for room and view for all users, add these lines to the bottom and alter accordingly.

$default_room = 1;
$default_view = "week";

**Comment by Langveisfra** [comment]
Some users solely are making bookings for one building, other users solely are making bookings for other buildings. It should be fine if the standard area shown (in our case buildings) was set for each user.

**Comment by campbell-m** [comment]
Yes, I can see that having user preferences would be a useful new feature.

I think the comment that @antham1616 was making was in response to your statement "Opening MRBS for the first time og re-opening the first page gives the area with the lowest index number".  By setting the defaults in the config file you can choose which area the users are directed to.  I realise that that's the same for all users and not what you are ideally looking for, but it might be better in the meantime than just having the lowest index number.

**Comment by Langveisfra** [comment]
I can se @antham1616's point. Thank you for a temporary solution!
